include/bx/platform.h

```diff
@@ -160,7 +160,7 @@
 #		define NOMINMAX
 #	endif // NOMINMAX
 //  If _USING_V110_SDK71_ is defined it means we are using the v110_xp or v120_xp toolset.
-#	if defined(_MSC_VER) && (_MSC_VER >= 1700) && (!_USING_V110_SDK71_)
+#	if defined(_MSC_VER) && (_MSC_VER >= 1700) && (!defined(_USING_V110_SDK71_) || (!_USING_V110_SDK71_))
```

Comment:
`!defined(_USING_V110_SDK71_)` would be sufficient.

Reply:
ok, i'll update the patch